### DISCUSSION TITLE
Error when I try to run `npx react-native run-android`

### DISCUSSION THREAD

**Discussion by BakungaBronson** [GaloyMoney/galoy-mobile#2269] [Q&A]
Hello, I went ahead and installed the requirements for the application and I setup my React Native environment. When I run `npx react-native run-android`, the "Installing the app" phase runs up to 100% but throws the following error.
![image](https://github.com/GaloyMoney/galoy-mobile/assets/51344005/5cc85186-2f54-4aec-8789-ed05f4199887)
I'm not sure what I'm doing wrong or if I missed anything. Thanks in advance for the help.

**Comment by nicolasburtey** [comment]
issue doesn't seem specific to this repo but on a dependency (react-native-vision-camera). have you check this repo to know if there are inputs on this problem?

**Reply by BakungaBronson** [reply]
Let me do just that, thank you.
